skrub/tests/test_table_vectorizer.py

```diff
@@ -682,6 +682,10 @@ def test_accept_pipeline():
     tv.fit(df)
 
 
+@pytest.mark.skipif(
```

Comment:
By checking properly the "polars" + sklearn, I found this issue as well.

Reply:
that's a good point -- it doesn't affect only this test but all the scikit-learn transformers. I'll try to see what would be involved to have polars support on scikit-learn versions that don't support it. If it's complicated I think at first we could say that  skrub only supports polars dataframe if you have a recent enough scikit-learn version installed

Reply:
this has been fixed by #941